src/dune_rpc_impl/server.ml

```diff
@@ -208,6 +208,7 @@ type t =
       (Dep_conf.t list * Build_outcome.t Fiber.Ivar.t) Job_queue.t
   ; watch_mode_config : Watch_mode_config.t
   ; mutable clients : Clients.t
+  ; build_count : int ref
```

Comment:
I think this can just be `; mutable build_count : int` and accessable via `t` in the request implementation.